components/style/themes/default.less

```diff
@@ -290,6 +290,10 @@
 @table-padding-vertical: 16px;
 @table-padding-horizontal: 8px;
 
+// Tag
+// --
+@tag-default-bg: @background-color-base;
```

Comment:
I'm fine with `@select-tag-bg` but why should `select-tag-bg` not default to the same color as `tag-default-bg` or vice versa. Just seems weird to have such similar variable for such similar things